src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -166,15 +169,14 @@ public MtAmazonDynamoDbBySharedTable(String name,
                                          String backupTablePrefix) {
         super(mtContext, amazonDynamoDb, meterRegistry, scanTenantKey, scanVirtualTableKey);
         this.name = name;
+        this.topLevelContext = topLevelContext;
         this.mtTableDescriptionRepo = mtTableDescriptionRepo;
         this.tableMappingCache = new MtCache<>(mtContext, tableMappingCache);
         this.tableMappingFactory = tableMappingFactory;
         this.partitioningStrategy = partitioningStrategy;
+        this.physicalTableManager = physicalTableManager;
         this.deleteTableAsync = deleteTableAsync;
         this.truncateOnDeleteTable = truncateOnDeleteTable;
-        this.mtTables = tableMappingFactory.getCreateTableRequestFactory().getPhysicalTables().stream()
```

Comment:
I think we don't need this because PhysicalTableManager already has a cache of the physical table descriptions that this server knows about (and that cache used to be in TableMappingFactory)